apps/src/code-studio/components/ExportDialog/Dialog.jsx

```diff
@@ -562,9 +619,10 @@ class ExportDialog extends React.Component {
     const needToSignIn =
       !isProjectLevel && signInState !== SignInState.SignedIn;
     const blockExport = userSharingDisabled || needToSignIn;
-    const blockText = userSharingDisabled
-      ? i18n.sharingBlockedByTeacher()
-      : i18n.createAccountToShareDescription();
+
+    if (blockExport) {
+      return this.renderWrappedInBaseDialog(this.renderBlocked());
```

Comment:
NIT: `this.renderBlocked()` reads like a boolean. Perhaps rename to something like `renderBlockedMessage()`